src/main/webapp/style.css

```diff
@@ -1,4 +1,4 @@
-#content {
+/*#content {
```

Comment:
Instead of commenting out this code, you can delete it.

It will always be in the git history in case we need to get it back :)